### PR TITLE
refactor: Cleanup references to old components in server release group

### PR DESCRIPTION
**Pull request by alexvy86**
## Description

- Remove some lines in alfred related to foreman, which had been removed before but were restored by mistake.
- Remove references to components / docker services that don't exist anymore (paparazzi, gateway, foreman).
- Update a TODO comment related to Paparazzi, which doesn't exist anymore.
- Cleanup VSCode launch targets (remove obsolete ones, adjust the names of others to match what's listening on that port).

## Reviewer Guidance

The review process is outlined on [this wiki page](https://github.com/microsoft/FluidFramework/wiki/PR-Guidelines#guidelines).

### CODE REVIEW

**Comment by Josmithr** [comment]
Nit: Since these changes affect actual code, I would recommend `refactor` instead of `chore` for the PR title. I also generally recommend against using `chore` as a rule, since it doesn't offer much semantic info.

File: server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts [hunk 1]
```diff
@@ -257,15 +257,6 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
...
 
-		// Foreman agent uploader does not run locally.
-		// TODO: Make agent uploader run locally.
-		const foremanConfig = config.get("foreman");
-		const taskMessageSender = services.createMessageSender(
-			config.get("rabbitmq"),
-			foremanConfig,
-		);
-		await taskMessageSender.initialize();
-
```
Comment (alexvy86) [inline server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts:268]: These were the ones [removed before](https://github.com/microsoft/FluidFramework/pull/15129) and then [re-added by mistake](https://github.com/microsoft/FluidFramework/pull/14627).

This seems to be the only use of `createMessageSender()` and thus of `RabbitmqTaskSender` inside it, which tracks with something @pradeepvairamani told me about RabbitMq not being in use anymore. But I see recent changes in `RabbitmqReceiver` which leave me unsure about also cleaning up all references (and code?) related to RabbitMq. @znewton the recent change there [was yours](https://github.com/microsoft/FluidFramework/pull/16368). Was that more of a "go through all floating promises in the repo and fix them" or does it actually mean that the class is still in use? The previous meaningful change to it is from 2021.

**Comment by alexvy86** [comment]
@hedasilv @znewton friendly ping :) . I feel decently confident that the changes here are safe, mostly want to see if anyone on the server team has more context about these things that might serve as confirmation, or be useful for further cleanup (I plan to tackle rabbitmq after this) and/or future reference.

File: server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts (hunk 1 above)
Comment (hedasilv) [reply server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts:268]: I think RabbitMQ is not in use indeed, and @znewton 's changes were like you said: "go through all floating promises and fix them". So I think it is safe to remove this.

**Review by hedasilv — APPROVED** [review]

**Comment by hedasilv** [comment]
> @hedasilv @znewton friendly ping :) . I feel decently confident that the changes here are safe, mostly want to see if anyone on the server team has more context about these things that might serve as confirmation, or be useful for further cleanup (I plan to tackle rabbitmq after this) and/or future reference.

Thanks for working on this, @alexvy86 ! I don't see any potential issues with this cleanup since we are not using these components, like you pointed out. In fact, I also did some cleanup on our private repo earlier this year where I removed most foreman-related configs.

**Pull request merged**